### PR TITLE
Move from maven to java -jar, deployment via tsuru app-deploy tsuru-a…

### PR DESCRIPTION
**Pull request by ajdevries**
…pp.jar

### CODE REVIEW

File: java8/start-jar [hunk 1]
```diff
@@ -8,5 +8,5 @@ XMX=${JAVA_MAX_MEMORY-128}
 MAXPERM=${JAVA_MAX_PERM-128}
 
 cd /home/application/current
-
-mvn clean package jetty:run
+java -XX:MaxPermSize=${MAXPERM}m -Xmx${XMX}m -jar tsuru-app.jar -Dserver.host=0.0.0.0 -Dserver.port=${PORT}
```
Comment (marcusaram) [inline java8/start-jar:11]: What if your filename is not tsuru-app.jar ?

Comment (ajdevries) [inline java8/start-jar:11]: Then it breaks. What is the alternative, check if file exists or pick the first .jar file?

Comment (marcusaram) [inline java8/start-jar:11]: Could be, but this will break with git type because it won't build anymore...

Comment (ajdevries) [inline java8/start-jar:11]: So what I want is a java -jar start, name of the jar doesn't matter, maybe the name of the app is a better alternative. What would your solution be?

Comment (ajdevries) [inline java8/start-jar:11]: Also added a gradle task. This only works for gradle projects. With this it is possible to use git push also. The only thing you have to do is add a stage tasks to your build.gradle. For ratpack the following example:

``` groovy
task stage() {
  dependsOn shadowJar
}

tasks.stage.doLast() {
  def name = "build/libs/${project.name}-${shadowJar.classifier}.jar"
  copy {
    from(name) {
      rename { 'current.jar' } 
    }
    into '.'
  }
}
```

See also https://devcenter.heroku.com/articles/deploying-gradle-apps-on-heroku perhaps a gradle platform should be created.

**Pull request merged**